src/store/modules/app.ts

```diff
@@ -83,9 +85,16 @@ const mutations = {
     }
   },
 
-  setTradeLiquidity(state: AppState, tradeLiquidity: LiquiditySelection): void {
+  setTradeLiquidity(state: AppState, tradeLiquidity: LiquiditySelection) {
     state.tradeLiquidity = tradeLiquidity;
     lsSet('tradeLiquidity', state.tradeLiquidity);
+  },
+
+  setSelectedPoolTokens(
+    state: AppState,
+    selectedPoolTokens: AppState['selectedPoolTokens']
+  ) {
+    state.selectedPoolTokens = selectedPoolTokens;
```

Comment:
I thought its better to just reuse the already defined type in `AppState` for all the params (also removed `void` - since its already inferred in TS)